tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -481,26 +481,35 @@ private:
          TVirtualCollectionProxy::DeleteTwoIterators_t fDeleteTwoIterators;
          TVirtualCollectionProxy::Next_t fNext;
       };
-      static RIteratorFuncs GetIteratorFuncs(TVirtualCollectionProxy *proxy, bool readOnly);
+      static RIteratorFuncs GetIteratorFuncs(TVirtualCollectionProxy *proxy, bool readFromDisk);
+
    private:
       class RIterator {
          const RCollectionIterableOnce &fOwner;
          void *fIterator = nullptr;
          void *fElementPtr = nullptr;
+
+         void Advance()
+         {
+            auto fnNext_Contig = [&]() {
+               auto &iter = *static_cast<unsigned char **>(fIterator), p = iter;
+               iter += fOwner.fStride;
+               return p;
+            };
+            fElementPtr = fOwner.fStride ? fnNext_Contig() : fOwner.fIFuncs.fNext(fIterator, fOwner.fEnd);
```

Comment:
> fnNext_Contig

At first glance this look like a typo (eyes want to read `fNext_Config`) )(second glance, I see it means `functionNextContiguous`).  

For regularity of reading consider this (or a variation thereof)
```
            auto contiguousNext = [](void *begin, size_t stride) {
               auto &iter = *static_cast<unsigned char **>(begin), p = iter;
               iter += stride;
               return p;
            };
            fElementPtr = fOwner.fStride ? contiguousNext(fIterator, fOwner.fStride) : fOwner.fIFuncs.fNext(fIterator, fOwner.fEnd);
```